src/components/v5/common/Fields/RadioButtons/RadioButtonsBase/RadioBase/RadioBase.tsx

```diff
@@ -18,7 +18,7 @@ const RadioBase: FC<RadioProps> = ({
     <input
       {...rest}
       value={value}
-      type="radio"
+      type="checkbox"
```

Comment:
probably a minor thing, but why isn't it a radio anymore?

Reply:
It's changed to checkbox to allow users to uncheck options. With radio it wasn't possible.